### PR TITLE
Create Binary Cross Entropy With Logits Loss.json

### PR DESCRIPTION
**Pull request by JessicaSun1022**

### CODE REVIEW

**Comment by sauravpandey123** [comment]
The calculations don't match with that should be gotten. For instance, for the following case:
Predicted: tensor([ 10.,  10., -10.])
Expected: tensor([0., 0., 1.])
Loss: 10.000045776367188 (using Python)
Yours gives 9.999.... , which isn't accurate as we have to match to 3 dp.
I'd recommend taking another look at the calculation, by comparing with the Binary Cross Entropy Loss function that is similar and already exists. 

image
